.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+<!--
+Thank you for submitting a PR to the rust rpm implementation!
+-->
+
+# PR Content Desc
+
+<!---
+DELETE all that do not apply:
+-->
+
+- 🩹 Bug Fix
+- 🦚 Feature
+- 📙 Documentation
+- 🦣 Legacy
+- 🪣 Misc
+
+<!---
+Mention the linked issue here.
+This will automatically close the issue once the PR is merged
+and creates a cross reference.
+-->
+
+Closes #
+
+## Changes proposed by this PR
+
+<!---
+Tell the reviewer WHAT changed, and WHY it had to change
+and how the WHAT and the WHY tie together.
+-->
+
+## Notes to reviewer
+
+<!---
+Leave a message to whoever is going to review this PR.
+Mainly, pointers to review the PR, and how they can test it.
+If things are still WIP or feedback on particular impl details
```

Comment:
"implementation" - might as well be explicit.

Reply:
```suggestion
If things are still WIP or feedback on particular implementation details
```